hermes-consumers/src/main/resources/application-integration.yml

```diff
@@ -4,4 +4,76 @@ consumer:
     readQueueCapacity: 1000
     waitBetweenUnsuccessfulPolls: true
     initialIdleTime: 10
-    maxIdleTime: 1000
\ No newline at end of file
+    maxIdleTime: 1000
+  batch:
+    poolableSize: 1024
+    maxPoolSize: 64 * 1024 * 1024
+    connectionTimeout: 500
+    connectionRequestTimeout: 500
+  commit:
+    offset:
+      period: 60
+      queuesSize: 200_000
+      queuesInflightDrainFullEnabled: false
+  rate:
+    limiterSupervisorPeriod: 30
+    limiterReportingThreadPoolSize: 30
+    limiterReportingThreadMonitoringEnabled: false
+    limiterHeartbeatModeDelay: 60
+    limiterSlowModeDelay: 60
+    convergenceFactor: 0.2
+    failuresNoChangeToleranceRatio: 0.05
+    failuresSpeedUpToleranceRatio: 0.01
+  sender:
+    async:
+      timeout:
+        http2:
+          milliseconds: 5_000
+          threadPoolSize: 32
+          threadPoolMonitoringEnabled: false
+  ssl:
+    enabled: true
+    protocol: TLS
+    keystoreSource: jre
+    keystoreLocation: classpath:client.keystore
+    keystorePassword: password
+    keystoreFormat: JKS
+    truststoreSource: jre
+    truststoreLocation: classpath:client.truststore
+    truststorePassword: password
+    truststoreFormat: JKS
+  http2:
+    client:
+      enabled: true
+      threadPoolSize: 10
+      threadPoolMonitoringEnabled: false
+      idleTimeout: 0
+      maxRequestsQueuedPerDestination: 100
+  http:
+    client:
+      connectionPoolMonitoringEnabled: false
+      requestQueueMonitoringEnabled: true
+      threadPoolSize: 30
+      threadPoolMonitoringEnabled: false
+      followRedirectsEnabled: false
+      maxConnectionsPerDestination: 100
+      idleTimeout: 0
+      maxRequestsQueuedPerDestination: 100
+
+kafka:
+  authorization:
+    enabled: false
+    mechanism: PLAIN
+    protocol: SASL_PLAINTEXT
+    username: username
+    password: password
+
+google:
```

Comment:
In my opinion this section should be a child of the `consumer` section. It applies neither to `hermes-frontend` nor `hermes-management`.

Reply:
OK, I see that this is not directly related to your PR. I will write a comment in #1508.